seaborn/distributions.py

```diff
@@ -129,6 +130,27 @@ def distplot(a, bins=None, hist=True, kde=True, rug=False, fit=None,
         >>> from scipy.stats import norm
         >>> ax = sns.distplot(x, fit=norm, kde=False)
 
+    Plot the distribution with a histogram, a KDE and fitted sum of 2
+    gaussians. This is a numerical algorithm (scipy.optimize.curve_fit), so a
+    reasonable initial guess of the parameters (p0 passed through fit_kws) is
+    important.
+
+    .. plot::
+        :context: close-figs
+
+        >>> def gauss(x, *p0):
+        ...     a, p0 = p0[0], p0[1:]
+        ...     return a*norm.pdf(x, *p0)
+        >>> def gauss2(x, *p0):
+        ...     p1 = p0[:2]
+        ...     p2 = p0[2:]
+        ...     return gauss(x, *p1)+gauss(x, *p2)
+        >>> a = np.append(np.random.normal(0.0, 0.5, size=500),
```

Comment:
I don't know how doctests work, but the error you get is because of this line and says:
```
======================================================================
FAIL: Doctest: seaborn.distributions.distplot
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/testenv/lib/python3.4/doctest.py", line 2187, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for seaborn.distributions.distplot
  File "/home/travis/build/mwaskom/seaborn/seaborn/distributions.py", line 39, in distplot
----------------------------------------------------------------------
File "/home/travis/build/mwaskom/seaborn/seaborn/distributions.py", line 148, in seaborn.distributions.distplot
Failed example:
    a = np.append(np.random.normal(0.0, 0.5, size=500),
Exception raised:
    Traceback (most recent call last):
      File "/home/travis/miniconda/envs/testenv/lib/python3.4/doctest.py", line 1318, in __run
        compileflags, 1), test.globs)
      File "<doctest seaborn.distributions.distplot[12]>", line 1
        a = np.append(np.random.normal(0.0, 0.5, size=500),
                                                          ^
    SyntaxError: unexpected EOF while parsing
```